### PR TITLE
fix(editor): Add Set up version control CTA

### PR DESCRIPTION
**Pull request by cstuncsik**

### CODE REVIEW

File: packages/editor-ui/src/components/MainSidebar.vue [hunk 1]
```diff
@@ -46,10 +46,7 @@
 							}}
 						</n8n-text>
 					</div>
-					<MainSidebarVersionControl
-						v-if="versionControlStore.preferences.connected"
-						:is-collapsed="isCollapsed"
-					/>
+					<MainSidebarVersionControl :is-collapsed="isCollapsed" />
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/MainSidebar.vue:49]: This doesn't seem to look right.. Can you double check? 

image

**Review by alexgrozav — CHANGES_REQUESTED** [review]
Something funky going on in the sidebar, but other than that looks good.

File: packages/editor-ui/src/components/MainSidebar.vue (hunk 1 above)
Comment (cstuncsik) [reply packages/editor-ui/src/components/MainSidebar.vue:49]: Fixed

**Review by alexgrozav — DISMISSED** [review]

**Review by alexgrozav — DISMISSED** [review]

**Review by flipswitchingmonkey — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@0.232.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.232.0)

**Pull request merged**